package.json

```diff
@@ -61,7 +29,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.0",
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@gnosis.pm/dapp-ui": "^0.5.3",
-    "@gnosis.pm/dex-js": "0.2.2",
+    "@gnosis.pm/dex-js": "0.3.0-RC.0",
```

Comment:
@alfetopito @W3stside should we close a version of dex-js?

Reply:
wondering if @Velenir wants to include his latest PR inside the latest version

Reply:
I think you are generating 0.4.0 and PR'in it to `release/1.0`, so solving this issue...